programs/github

```diff
@@ -134,24 +128,26 @@ if action == 'clone' then
 end
 
 if action == 'auth' then
```

Comment:
nitpick: add `action = validActions[action]` up where actions is defined, then compare against `validActions.auth`. Then nothing else needs to know about the `validActions` table, and `getFlagTable()` becomes pointless